.github/workflows/gem_notifications.yml

```diff
@@ -36,3 +36,17 @@ jobs:
             redis
             resque
             unicorn"
+
+  cve_notifications:
```

Comment:
The benefit of going with a separate entry here is that it will run in parallel with the gem notification job and both won't be impacted by an issue with the other.

The drawback is with cost and more moving parts (for example, there are now two separate internet calls to fetch 'httparty)'.

We'll want to give some thought to these trade-offs as we continue to add more functionality to Gabbi.

Some thoughts...

- Maybe a single entrypoint invokes Gabbi and it goes through a list of work to do.
    - Each separate task will need to be hardened against crashing Gabbi itself, so that the following tasks are guaranteed to run
- Once Gabbi itself is a gem, a single `gem install gabbi` will simplify the fetching of all of Gabbi's own dependencies
- If gem fetching over the internet ever becomes an issue, we can always switch to GHA caching or prebuilt and stored `.gem` files
- Design-wise, we'll want to think of mitigating unhappy path scenarios. Knowing about a CVE is pretty important. Currently if the Gabbi run to discover them fails, there's a chance the CVE won't be picked up by the next Gabbi run 24 hours later and we'll never see the notification.

(Just thoughts to bear in mind as Gabbi continues to evolve - no related requests for change for these things with this PR)